server/mpm/event/event.c

```diff
@@ -2249,7 +2303,6 @@ static void * APR_THREAD_FUNC listener_thread(apr_thread_t * thd, void *dummy)
         }
     } /* listener main loop */
 
-    close_listeners(&closed);
```

Comment:
Why is this no longer needed?

Reply:
Single point of exit in the listener loop now, and close_listeners() is already called there.

Reply:
ok